doc/classes/TileSet.xml

```diff
@@ -44,6 +44,8 @@
 			<argument index="1" name="neighbor_id" type="int">
 			</argument>
 			<description>
+				Determines when the auto-tiler should consider two different auto-tile IDs to be bound together.
+				Also, neighbor_id can (and will) be -1 when checking a tile against air.
```

Comment:
Use `[code]` tags to reference parameters and values:

```suggestion
				Also, [code]neighbor_id[/code] can (and will) be [code]-1[/code] when checking a tile against air.
```

Also, can't we simplify "can (and will)" to just "will"?